src/api/sso/src/middleware.js

```diff
@@ -106,6 +106,7 @@ const createNewProfile = async (id, body) => {
     githubUsername,
     githubAvatarUrl,
     blogUrl,
+    youtubeUrl,
```

Comment:
I don't think we need to modify the schema of a user in the DB.  A YouTube channel is just another RSS feed that happens to be hosted on YouTube.